### ISSUE TITLE
Text task with suggestions should be marked up as a combobox text input

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#3031]
## Package
lib-classifier

## Is your feature request related to a problem? Please describe.
While writing tests for the text task with suggestions, I couldn't use accessible names and roles for the text input and list of suggestions, because they aren't present.

## Describe the solution you'd like
When the text input has a list of suggested values, it should follow the [WAI-ARIA 1.2 design pattern for a combobox with manual autocomplete](https://www.w3.org/TR/wai-aria-practices-1.2/examples/combobox/combobox-autocomplete-list.html).

## Additional context
https://a11ysupport.io has [a comprehensive suite of tests](https://a11ysupport.io/tests/apg__aria-1-2-combobox-with-list-autocomplete-example) which show that the latest WAI-ARIA pattern has good support.

I've opened a Grommet PR to add the appropriate ARIA roles and states to the `TextInput` component when suggestions are present.
https://github.com/grommet/grommet/pull/6079

**Comment by eatyourgreens** [comment]
Grommet 2.24 is out now, which includes this.

**Comment by eatyourgreens** [comment]
Closed by #3260.
